DEVELOPMENT.md

```diff
@@ -142,6 +142,7 @@ while [[ $(kubectl get crd gateways.networking.istio.io -o jsonpath='{.status.co
   echo "Waiting on Istio CRDs"; sleep 1
 done
 kubectl apply -f "https://raw.githubusercontent.com/knative-sandbox/net-istio/master/third_party/${STABLE_VERSION}/istio-minimal.yaml"
+cd serving
```

Comment:
We probably don't need this command as we assume that you are already under `serving/` directory.

Reply:
Since I jumped in directly to that section, I ran the first commands from a different directory and then I was not sure if the last command had some sort of mistake in it. I would have found the directory change helpful if included, hence the update. But if the deployment document is viewed as something that should be consumed as a whole, I can close this PR.

Reply:
yea, we don't explicitly tell developers to be under `/serving` directory. I think we can add the instruction of entering `serving/` directory in the beginning of https://github.com/knative/serving/blob/master/DEVELOPMENT.md#starting-knative-serving as the following installations including istio, certmangaer, knative, etc are also based on this assumption.

Reply:
I like that idea. Should I update this PR or close it?

Reply:
You can just update it within this PR. thanks!